app/src/androidTest/java/com/github/orkest/ViewModel/DeezerApiIntegrationTest.kt

```diff
@@ -27,6 +28,24 @@ class DeezerApiIntegrationTest {
         Assert.assertEquals("Booba",value.data[0].artist.name)
     }
 
+    /**
+     * tested
+     */
+    fun JsonResponseforUSerId(){
+        val value = DeezerApiIntegration().addANewSongToOrkestPlayList("frsizHK2HlrzljGuf9s09gV0sF1Wbf7fWCnQe4w1wFbDI0zCkqV","11300267884","1999525067").get()
+        Log.d("HELLO", value.toString())
+
+    }
+
+    /**
+     * tested
+     */
+    fun createPlaylist(){
+        val value = DeezerApiIntegration().createANewPlaylistOnTheUserProfile("2297625024","frsizHK2HlrzljGuf9s09gV0sF1Wbf7fWCnQe4w1wFbDI0zCkqV").get()
+        Log.d("HELLO", value.toString())
+
+    }
+
```

Comment:
JsonResponseforUSerId should be renamed jsonResponseForUserId for best practices. Also "value" is a meaningful name, same for createPlaylist

Also, the comment block above the function should be more precise, same for createPlaylist

Reply:
I understand the code is not perfect but it will be fix on a next issue